strings.txt

```diff
@@ -16217,48 +16217,644 @@
+    de = Ihre Änderungsvorschläge für die Karte werden an OpenStreetMap gesendet: Beschreiben Sie die Details der Objekte, die Sie in MAPS.ME nicht bearbeiten können.
+    fi = Ehdottamasi muutokset lähetetään OpenStreetMap-yhteisöön. Kuvaa tiedot, joita ei voi muokata MAPS.ME:ssä
+    cs = Vámi navržené změny odešleme do komunity OpenStreetMap. Popište podrobnosti, které nelze upravit v MAPS.ME.
+    it = Le modifiche suggerite saranno inviate alla community OpenStreetMap. Descrivere i dettagli che non è possibile modificare in MAPS.ME
+    nb = De foreslåtte endringene vil sendes til OpenStreetMap-gruppen. Beskriv detaljene som ikke kan redigeres i MAPS.ME
+    zh-Hans = 您建议的更改将发送至 OpenStreetMap 社区。说明无法在 MAPS.ME 编辑的详情。
+    th = คำแนะนำการเปลี่ยนแปลงของคุณจะถูกส่งไปยังกลุ่ม OpenStreetMap โปรดอธิบายรายละเอียดที่ MAPS.ME ไม่สามารถแก้ไขได้
+    ja = あなたが提案した変更は、OpenStreetMapのコミュニティに送信されます。MAPS.MEで編集できない詳細を説明してください。
+    ro = Modificările sugerate vor trimise către comunitatea OpenStreetMap. Descrieți detalii ce nu pot fi adăugate în be the details which cannot be edited in MAPS.ME
+    ar = سيتم إرسال التغييرات التي اقترحتها إلى مجتمع OpenStreetMap. اذكر التفاصيل التي لا يمكن تحريرها في MAPS.ME
+    sk = Vami navrhované zmeny sa odošlú do komunity OpenStreetMap. Popíšte detaily, ktoré sa nedajú upraviť v MAPS.ME
+    es = Los cambios que ha sugerido se enviarán a la comunidad de OpenStreetMap. Describa los detalles que no pueden editarse en MAPS.ME.
+    pl = Twoje sugestie zmian zostaną wysłane do społeczności OpenStreetMap. Opisz szczegóły, których nie można edytować w MAPS.ME
+    nl = Uw voorgestelde wijzigingen worden verzonden naar de OpenStreetMap-community. Beschrijf de details die niet kunnen worden bewerkt in MAPS.ME
 
   [editor_more_about_osm]
     tags = ios, android
     en = More about OpenStreetMap
+    ru = Подробнее об OpenStreetMap
+    fr = En savoir plus sur OpenStreetMap
+    da = Mere om OpenStreetMap
+    id = Selengkapnya tentang OpenStreetMap
+    ko = OpenStreetMap 정보
+    sv = Mer om OpenStreetMap
+    tr = OpenStreetMap hakkında ek bilgi
+    uk = Більше про OpenStreetMap
+    vi = Thông tin bổ sung về OpenStreetMap
+    hu = További részletek az OpenStreetMap-ról
+    de = Mehr Informationen über OpenStreetMap
+    fi = Lisätietoja OpenStreetMap:ista
+    cs = Více o projektu OpenStreetMap
+    it = Ulteriori informazioni su OpenStreetMap
+    nb = Mer om OpenStreetMap
+    zh-Hans = 关于 OpenStreetMap 的更多信息
+    th = ข้อมูลเพิ่มเติมเกี่ยวกับ OpenStreetMap
+    ja = OpenStreetMapについての詳細
+    ro = Mai multe despre OpenStreetMap
+    ar = المزيد عن OpenStreetMap
+    sk = Viac o OpenStreetMap
+    es = Más acerca de OpenStreetMap
+    pl = Więcej o OpenStreetMap
+    nl = Meer over OpenStreetMap
+
+  [editor_operator]
+    comment = Строка в редакторе
+    en = Operator
+    ru = Владелец
```

Comment:
Вроде было именно так. Кастую Игоря в чат.

> On Apr 26, 2016, at 3:43 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61078040>:
> 
> > +    it = Ulteriori informazioni su OpenStreetMap
> > +    nb = Mer om OpenStreetMap
> > +    zh-Hans = 关于 OpenStreetMap 的更多信息
> > +    th = ข้อมูลเพิ่มเติมเกี่ยวกับ OpenStreetMap
> > +    ja = OpenStreetMapについての詳細
> > +    ro = Mai multe despre OpenStreetMap
> > +    ar = المزيد عن OpenStreetMap
> > +    sk = Viac o OpenStreetMap
> > +    es = Más acerca de OpenStreetMap
> > +    pl = Więcej o OpenStreetMap
> > +    nl = Meer over OpenStreetMap
> > +
> > +  [editor_operator]
> > +    comment = Строка в редакторе
> > +    en = Operator
> > +    ru = Владелец
> Оператор и владелец? По-моему, один из вариантов некорректный.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295#r61078040>